backend/src/main/java/com/roc/app/user/User.java

```diff
@@ -0,0 +1,88 @@
+package com.roc.app.model;
+
+import jakarta.persistence.*;
+
+import java.util.Objects;
+
+@Entity
+@Table(name = "\"user\"")
```

Comment:
After script update, an update here will also be needed.
```suggestion
@Table(name = "users")
```